docs/runtimes/mlrun_jobs.ipynb

```diff
@@ -955,19 +926,19 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Pipeline results are stored at the `artifact_path` location:"
+    "Pipeline results are stored at the `artifact_path` location. By default, MLRun adds ```/{{workflow.uid}}``` to the path."
```

Comment:
It's a bit different than that - the artifact path for workflows can be either:
1. The artifact_path as passed to the specific call for `run()`, as shown below. In this case, MLRun does not modify the user-provided path
2. The project's artifact_path (set by `project.spec.artifact_path = '<some path>'`)
3. MLRun's default artifact-path, if set.

In the 2nd and 3rd option MLRun will add `/{{workflow.uid}}` to the path as described, but only if the path does not already contain it.